app/plugins/router.js

```diff
@@ -37,6 +35,13 @@ if (config.endemics.enabled) {
     require('../routes/endemics/pi-hunt-recommended').handlers,
     require('../routes/endemics/pi-hunt-all-animals').handlers
   )
+  if (!config.multiSpecies.enabled) {
```

Comment:
Would normally expect a [code warning about dynamic requires](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-dynamic-require.md), might be better to just require everything and then use the logic for whether they make it into the routes collection?

Reply:
Yeah agree that's a better pattern, but there's a couple of things at play here:

- hopefully we'll move this to ESM soon and just have import statements and this won't even be a thing
- Obviously I followed the existing pattern here which you can argue is not the right thing to always do but it's also done lots of other places in this repo and I'm not going to solve them all in this PR, so 'one more' doesn;t hurt til we wait for the above.

having said that if you feel really strongly I can go back and change it, it's not a life changer either way

Reply:
(and have)